handler.go

```diff
@@ -113,10 +125,11 @@ func handlerByMethod(route *string, do interface{}, method string) *handler {
 			if ok {
 				h.formParams[key] = i
 			}
-		}
-
-		for i := 0; i < userDefinedT.NumField(); i++ {
-			_, ok := userDefinedT.Field(i).Tag.Lookup("json")
+			key, ok = userDefinedT.Field(i).Tag.Lookup("query")
+			if ok {
+				h.queryParams[key] = i
+			}
+			_, ok = userDefinedT.Field(i).Tag.Lookup("json")
```

Comment:
if struct like this
```
type User struct {
  name string `json:"name"`
  id       int       `query:"id"`
}
```
expectJsonRequest will be false, is that you want?

Reply:
I missing myself meaning XD.

Reply:
fixed at: 1488a30